packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -465,8 +465,14 @@ export const run = async () => {
     await createMetricsServer({
       logger,
       registry: metrics.registry,
-      port: config.uint('prometheus-port', 7300),
-      hostname: config.str('prometheus-hostname', '127.0.0.1'),
+      port: config.uint(
+        'prometheus-port',
```

Comment:
Can we rename `prometheus-port` -> `metrics-port` and `prometheus-hostname` -> `metrics-address`?

Should these env name be separated by package? Something like  `BATCHSUBMITTER_METRICS_PORT`  and `BATCHSUBMITTER_METRICS_ADDRESS`?

Reply:
The prefix `BATCH_SUBMITTER` is supported based on this line here: https://github.com/ethereum-optimism/optimism/blob/cc209be13ccefc9d0c376413c81cf39be04d8f05/packages/batch-submitter/src/exec/run-batch-submitter.ts#L76. Whatever name is passed to the config object will parse out prefixed env vars where the name is the prefix

A library called [bcfg](https://github.com/bcoin-org/bcfg) is used for env var parsing. It is lightweight, simple and has no dependencies besides the JS std library. It parses prefixed env vars, CLI args and can read from a config file at `~$HOME/.<name-passed-to-config>/*` with the `config.open` function - https://github.com/bcoin-org/bcfg/blob/1216c775832c2e138ae6451912a27ba496dc4386/lib/config.js#L128

Reply:
Oh, nice! LGTM then

Reply:
This still doesn't address the change to `metrics-*`

Reply:
^renamed!